README.md

```diff
@@ -295,6 +295,12 @@ No version will be set by Builder. **(not recommended)**
 > Not recommended unless you generate a new version in a pre-commit hook. Manually
 > setting versions is error-prone.
 
+#### androidVersionCode
+
+Configure the android `versionCode`.
+
+When not specified, the version code is generated from the version using the `major * 1000000 + minor * 10000 + patch` scheme;
```

Comment:
probably we need to make it possible to have more than 99 minor versions, since it's may not be very uncommon in some use cases.